spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/convert/ConverterAwareMappingSpannerEntityWriter.java

```diff
@@ -193,6 +193,81 @@ public SpannerWriteConverter getSpannerWriteConverter() {
 		return this.writeConverter;
 	}
 
+	/**
```

Comment:
@dmitry-s  Don't be alarmed by these big new sections .They're just moved up because now they're public.